.gitignore

```diff
@@ -17,3 +17,4 @@ TAGS
 /haskell/test/resources/stablecoin.tz
 /haskell/test/resources/stablecoin.fa1.2.tz
 /haskell/test/resources/metadata.tz
+/haskell/test/resources/sources.json
```

Comment:
Here too the comment of this "group" doesn't match this addition:
> We also ignore them in `haskell/test/resources/` because they are symlinked for tests.

I think a new comment to explain why we need this (or rather why it is there) would be the best

Reply:
Done